LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Nathan Rajlich
```

Comment:
This seems incomplete, Nathan has committed not only in 2017, and there have been other contributors as well...

Reply:
Ah ok

Reply:
Could someone then provide a valid LICENSE.
GitHub License check fails/errors without it.

Reply:
Just update it to `2012 Nathan Rajlich, Linus Unnebäck and contributors` and it LGTM